pkg/sdk/api_integrations_def.go

```diff
@@ -150,6 +150,7 @@ var ApiIntegrationsDef = g.NewInterface(
 			Show().
 			SQL("API INTEGRATIONS").
 			OptionalLike(),
+		g.ResourceIDHelperMethod,
```

Comment:
Resource is bad, because it's coupled with TF resource, and SDK should not be coupled with TF. My proposals are `g.ShowObjectIdMethod` and `g.ObjectIdMethod`. I like the former more.

Reply:
g.ShowObjectIdMethod and g.ShowObjectTypeMethod seems fine to me as well